go.mod

```diff
@@ -3,3 +3,5 @@ module github.com/rogpeppe/go-internal
 go 1.11
 
 require gopkg.in/errgo.v2 v2.1.0
+
+go 1.13
```

Comment:
Hmm how did this get added given the `go.mod` already has `go 1.11`?

Reply:
good point. must've been the rebase. fixed.